index.js

```diff
@@ -41,9 +41,12 @@ async function tryPayInvoice({ invoice, paymentAmountSats, maxRouteFeePpm, outCh
     console.log(`Payment confirmed, with fee ${paymentResult.safe_fee} satoshis, and ppm ${feePpm}`)
 }
 
-async function attemptPaymentToDestination({ destination, outChannelIds }) {
+async function attemptPaymentToDestination({ destination, outChannelIds, attempt }) {
     let invoice
-    const paymentAmountSats = destination.PAYMENT_AMOUNT_SATS
+    const paymentAmountSats = destination.PAYMENT_AMOUNT_SATS > 0 ? destination.PAYMENT_AMOUNT_SATS : destination.MAX_PAYMENT_AMOUNT_SATS
+    if (attempt > 0) {
+        paymentAmountSats = paymentAmountSats*Math.pow((1/destination.PAYMENT_MULTIPLIER),attempt)
+    }
```

Comment:
Would `const` need to become `var`? According to [ES6 (2015)](https://www.w3schools.com/JS/js_es6.asp) consts cant be re-assigned.

Reply:
Also
```js
paymentAmountSats *= Math.pow ( ( 1 / destination.PAYMENT_MULTIPLIER ), attempt )
```